dask/dataframe/io/tests/test_parquet.py

```diff
@@ -3342,3 +3342,42 @@ def test_roundtrip_rename_columns(tmpdir, engine):
     df1.columns = ["d", "e", "f"]
 
     assert_eq(df1, ddf2.compute())
+
+
+def test_filtered_column_subset(tmpdir, engine):
+    df = pd.DataFrame({"col": range(20), "part": ["A", "B"] * 10})
+    ddf = dd.from_pandas(df, npartitions=2)
+    ddf.to_parquet(tmpdir, partition_on=["part"], engine=engine)
+
+    # # Filtered on a partition column
+    ddf2 = dd.read_parquet(tmpdir, filters=[[("part", "=", "A")]], engine=engine)
```

Comment:
```suggestion
    ddf2 = dd.read_parquet(tmpdir, filters=[("part", "==", "A")], engine=engine)
```

"fastparquet" cannot handle DNF for partitioned columns (I just added a note about this in #7091 ).  Also, "fastparquet" seems to require `"=="`.